src/components/ProcessDocumentScreen/ProcessDocumentScreen.tsx

```diff
@@ -63,157 +68,33 @@ export const ProcessDocumentScreen: FunctionComponent<ProcessDocumentScreen> = (
-                        <div className="text-xs text-cloud-500 ml-1">({size})</div>
-                      </div>
-                    );
-                  })}
-                </div>
-                <div className="flex py-4">
-                  <div className="col-auto ml-auto">
-                    <Button
-                      className="bg-white text-cerulean hover:bg-cloud-100 mb-4"
-                      data-testid="download-fail-button"
-                      onClick={() => {
-                        generateZipFile(failedProcessedDocuments[0].documents);
-                      }}
-                    >
-                      <div className="flex">
-                        <Download />
-                        <div className="text-cerulean ml-2">Download Failed Documents</div>
-                      </div>
-                    </Button>
-                  </div>
-                </div>
-              </div>
-            )}
-            {queueState === QueueState.ERROR && (
-              <div className="bg-red-100 px-8 py-6 rounded-xl">
-                <div className="mb-8">
-                  <div className="text-rose font-bold mb-8" data-testid="error-title">
-                    {`The document(s) could not be ${isIssuingFlow ? "published" : "revoked"} at this time.`}
-                  </div>
-                  <div>
-                    Please contact TradeTrust via email or client representative to resolve your issue. Alternatively,
-                    please try again.
-                  </div>
-                </div>
-                <Button className="bg-white hover:bg-cloud-100 flex mx-auto">
-                  <a
-                    download={generateFileName({
-                      network: config?.network,
-                      fileName: "error-log",
-                      extension: "txt",
-                      hasTimestamp: true,
-                    })}
-                    href={`data:text/plain;charset=UTF-8,${JSON.stringify(error, null, 2)}`}
-                  >
-                    <div className="flex">
-                      <Download />
-                      <div className="text-cerulean ml-2">Download Error Log</div>
-                    </div>
-                  </a>
-                </Button>
-              </div>
-            )}
+            <ProcessDocumentTitle
+              queueState={queueState}
+              successfulDocumentsCount={successfulProcessedDocuments.length}
+              failedDocumentsCount={failedProcessedDocuments.length}
+              type={type}
+            />
           </div>
+          <ProcessDocumentContent
+            network={config?.network}
+            queueState={queueState}
+            queueType={type}
+            failedDocuments={failPublishedDocuments}
+            pendingDocuments={pendingProcessDocuments}
+            successDocuments={successfulProcessedDocuments}
+            fileName={fileName || ""}
+            downloadErrorName={generateFileName({
```

Comment:
actually i not sure which is better:

- default "" at prop pass here?
- or default at component level -> https://github.com/TradeTrust/document-creator-website/blob/21fba047146c40e3f0ac2ee56ba7ea3dd71154bb/src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx#L23

wdyt?

Reply:
really not sure tbh, depends on existing convention which i'm not super familiar with